ios/RCTTWVideoModule.m

```diff
@@ -188,7 +188,33 @@ - (void)addParticipantView:(TVIVideoView *)view sid:(NSString *)sid trackSid:(NS
 
 RCT_REMAP_METHOD(setLocalVideoEnabled, enabled:(BOOL)enabled setLocalVideoEnabledWithResolver:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject) {
-  [self.localVideoTrack setEnabled:enabled];
+  if (self.localVideoTrack != nil) {
+    [self.localVideoTrack setEnabled:enabled];
+  }
+  if (self.room == nil) {
+    // No room to add/remove tracks from, so ignore.
+    return;
```

Comment:
You'll want to make sure to either call resolve or reject otherwise any JS promises will not progress.

Reply:
Fixed, thank you